### DISCUSSION TITLE
How to uninstall Ant Media?

### DISCUSSION THREAD

**Discussion by Mohit-3196** [ant-media/Ant-Media-Server#4921] [FAQs] [answered]
I have installed Ant Media as a service on my Ubuntu machine. I want to uninstall it from the current machine and shift it to another one.
I can't find any instructions about uninstalling it. How can I uninstall AMS?

Thank you

**Comment by yashtandon113** [comment, marked as answer]
Here are the commands to completely uninstall Ant Media Server:

```
sudo service antmedia stop
sudo rm -rf /usr/local/antmedia
sudo userdel antmedia
sudo systemctl reset-failed
```
